src/SFA.DAS.RoatpGateway.Web/ViewModels/RoatpGatewayPageViewModel.cs

```diff
@@ -35,5 +35,8 @@ public class RoatpGatewayPageViewModel
         public string Heading { get; set; }
         public string Caption { get; set; }
         public string NoSelectionErrorMessage { get; set; }
+    
+        public string ClarificationAnswer { get; set; }
+        public DateTime? DateOfClarificationCheck { get; set; }
```

Comment:
not sure about naming. Isn't this when Clarification is Requested, or is it something else (like when it was recieved)?

Reply:
that's a leftover from refactoring, I'm removing it